VERSION

```diff
@@ -83,15 +83,23 @@ date="Unreleased developer copy"
 # format.
 
 libmpi_so_version=12:3:0
+# was 12:2:0
 libmpi_cxx_so_version=2:3:1
+# was 2:3:1
 libmpi_mpifh_so_version=12:1:0
+# was 12:0:0
 libmpi_usempi_tkr_so_version=6:0:1
-libmpi_usempi_ignore_tkr_so_version=7:0:1
+libmpi_usempi_ignore_tkr_so_version=8:0:2
+# was 7:0:1
```

Comment:
I understand that - my understanding from the libtool docs is that a change to the APIs requires a change to the "current" component, and that must in turn reset "revision" and "age" to 0.